rurusetto/wiki/serializers.py

```diff
@@ -8,5 +8,5 @@ class RulesetSerializer(serializers.HyperlinkedModelSerializer):
     """
     class Meta:
         model = Ruleset
-        fields = ['id', 'name', 'slug', 'description', 'icon', 'logo', 'cover_image', 'opengraph_image', 'source', 'content',
+        fields = ['id', 'name', 'slug', 'description', 'icon', 'dark_icon', 'logo', 'cover_image', 'opengraph_image', 'source', 'content',
```

Comment:
```suggestion
        fields = ['id', 'name', 'slug', 'description', 'icon', 'light_icon', 'logo', 'cover_image', 'opengraph_image', 'source', 'content',
```
Because the website's default theme is dark so it need to change.